wd-java-testng/src/test/java/com/example/InventoryTest.java

```diff
@@ -47,6 +46,11 @@ void checkInventoryPageLooksTheSame() {
         var inventoryPage = new InventoryPage(driver);
         inventoryPage.open();
 
+        // this is here to generate a diff to better demonstrate sauce visual
+        if(System.getProperty("modified") != null) {
```

Comment:
```suggestion
        if(dotenv.get("modified") != null) {
```

Reply:
this needs to be set on the terminal by end users and setting it on the terminal is different between windows and mac. so going for system properties was more straight forward, does this make sense?

Reply:
yes, sorry for the confusion